contrib/cirrus/lib.sh

```diff
@@ -359,6 +374,9 @@ remove_packaged_podman_files(){
         if [[ -d "$fullpath" ]] || [[ $(basename "$fullpath") == "conmon" ]] ; then continue; fi
         ooe.sh sudo rm -vf "$fullpath"
     done
+
+    # Be super extra sure and carefuling vs performant and completely safe
```

Comment:
"carefuling"? Could you tweak this comment a bit? I'm not really understanding it.

Reply:
Oh sorry, it's an old meme.  Sometimes slips out when I'm wielding heavy machinery.

Reply:
okay fixed

Looked but couldn't find references to the origin of 'carefuling'...it was mistakenly printed on the back of a particular neon-yellow safety jacket sold for a few $$$ at discount stores, all `Made in $SOME_COUNTRY` naturally.  Old grey-beard grease-monkeys bought them up to wear on the job and some made their way into You Tube videos.  The irony of the whole thing makes me chuckle and cry at the same time.